homeassistant/components/trafikverket_weatherstation/diagnostics.py

```diff
@@ -0,0 +1,17 @@
+"""Diagnostics support for Nord Pool."""
```

Comment:
Stale

Reply:
LOL. Always forget to rename when I steal some stuff 🤣